Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -429,6 +449,24 @@ void SpreadsheetWidget::redo()
     update();
 }
 
+void SpreadsheetWidget::change_cell_static_color_format(bool background)
+{
+    VERIFY(m_selected_view);
+    auto* sheet_ptr = m_selected_view->sheet_if_available();
+    VERIFY(sheet_ptr);
+    auto& sheet = *sheet_ptr;
```

Comment:
This is kind of clunky, you should be able to do that in just one line (including the VERIFY).

Reply:
Done in f5101cf